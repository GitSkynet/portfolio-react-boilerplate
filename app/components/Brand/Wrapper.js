import styled from 'styled-components';

const Wrapper = styled.div`
  padding: 0.5rem;
  padding-bottom: 0;
`;

export default Wrapper;
