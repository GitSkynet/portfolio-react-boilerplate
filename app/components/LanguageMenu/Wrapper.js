import styled from 'styled-components';

const Wrapper = styled.div`
  align-self: center;
  position: relative;
`;

export default Wrapper;
