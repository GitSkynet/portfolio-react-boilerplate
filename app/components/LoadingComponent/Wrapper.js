import styled from 'styled-components';

const Wrapper = styled.article`
  align-self: center;
  margin: 2em auto;
  position: relative;
  width: 100%;
  text-align: center;
`;

export default Wrapper;
