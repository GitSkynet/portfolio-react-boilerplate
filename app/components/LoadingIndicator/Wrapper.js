import styled from 'styled-components';

const Wrapper = styled.div`
  align-self: center;
  height: 40px;
  margin: 2em auto;
  position: relative;
  width: 40px;
`;

export default Wrapper;
