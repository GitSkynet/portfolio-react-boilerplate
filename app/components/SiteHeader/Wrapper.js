import styled from 'styled-components';

const Wrapper = styled.div`
  display: flex;
  width: 100%;
  border-bottom: 0.5vh solid #465a65;
`;

export default Wrapper;
