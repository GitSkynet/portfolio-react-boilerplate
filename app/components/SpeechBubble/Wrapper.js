import styled from 'styled-components';
import { rem } from 'polished';

import Bubble from './Bubble';

const Wrapper = styled.div`
  margin-bottom: 9vh;
  padding: 1vh 1vw;
  width: 100%;
  filter: drop-shadow(0 0 10px grey);

  @media (min-width: 1008px) {
    max-width: ${rem('1008px')};
  }

  ${Bubble}

  p {
    &:not(:first-child) {
      margin-top: 1.5rem;
    }
  }
`;

export default Wrapper;
