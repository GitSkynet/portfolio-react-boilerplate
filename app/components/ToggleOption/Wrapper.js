import styled from 'styled-components';
import { rem } from 'polished';

const Wrapper = styled.div`
  padding: ${rem(7)};
  background-color: #797978;
`;
export default Wrapper;
