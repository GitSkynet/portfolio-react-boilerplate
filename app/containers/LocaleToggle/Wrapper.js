import styled from 'styled-components';

const Wrapper = styled.aside`
  margin-left: auto;
`;

export default Wrapper;
