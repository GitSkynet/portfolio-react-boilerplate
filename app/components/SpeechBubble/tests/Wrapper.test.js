import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

import Wrapper from '../Wrapper';

describe('<Wrapper />', () => {
  it('Should render and match the snapshot', () => {
    const wrapper = shallow(<Wrapper />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
