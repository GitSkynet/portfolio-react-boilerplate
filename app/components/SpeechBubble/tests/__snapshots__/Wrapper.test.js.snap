// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> Should render and match the snapshot 1`] = `
.c0 {
  margin-bottom: 9vh;
  padding: 1vh 1vw;
  width: 100%;
}

.c0 .Bubble__Bubble-kfMqWCp:not(:first-child) {
  margin-top: 1.5rem;
}

@media (min-width:1008px) {
  .c0 {
    max-width: 63rem;
  }
}

<div
  className="c0"
/>
`;
